prepare-image.sh

```diff
@@ -32,6 +32,12 @@ if [[ $INSTALL_TYPE == "source" ]]; then
 
     python3 -c 'import os; import sys; import jinja2; sys.stdout.write(jinja2.Template(sys.stdin.read()).render(env=os.environ, path=os.path))' < "${IRONIC_PKG_LIST}" > "${IRONIC_PKG_LIST_FINAL}"
 
+    if [ -n $SUSHY_SOURCE ]; then
+        curl -L ${UPPER_CONSTRAINTS_FILE:-"https://releases.openstack.org/constraints/upper/master"} -o /tmp/sushy-constraints.txt
+        UPPER_CONSTRAINTS_FILE="/tmp/sushy-constraints.txt"
```

Comment:
If you put this on line up you could use it twice

Reply:
Not really, then user-provided UPPER_CONSTRAINTS_FILE won't work.

Reply:
ack, I missed that